docs/examples/Multi_Tracker_Example.py

```diff
@@ -0,0 +1,558 @@
+# whereas the ESIF has very poor relative performance. Caution should be taken when using the
+# ESIF in practice. A singular performance metric is calculated from this by summing the
+# OSPA value over all timesteps:
+
+# sum up distance error from ground truth over all timestamps
+ospa_EKF_total = 0
+ospa_UKF_total = 0
+ospa_P_total = 0
+ospa_ESIF_total = 0
+for i in range(0, len(ospa_metric_EKF.value)):
+    ospa_EKF_total += ospa_metric_EKF.value[i].value
+    ospa_UKF_total += ospa_metric_UKF.value[i].value
+    # Same length as EKF so can do calculation in the same loop
+
+for i in range(0, len(ospa_metric_P.value)):
+    ospa_P_total += ospa_metric_P.value[i].value
+
+for i in range(0, len(ospa_metric_ESIF.value)):
+    ospa_ESIF_total += ospa_metric_ESIF.value[i].value
+
+print("OSPA total value for EKF is ", f'{ospa_EKF_total:.3f}')
+print("OSPA total value for UKF is ", f'{ospa_UKF_total:.3f}')
+print("OSPA total value for particle filter is ", f'{ospa_P_total:.3f}')
+print("OSPA total value for ESIF is ", f'{ospa_ESIF_total:.3f}')
+
+# %%
+# Finally, we calculate the SIAP metrics for the different trackers. Here we only display
+# the metrics for the EKF, but the user can uncomment the other sections of code for full metrics:
+from stonesoup.metricgenerator.metrictables import SIAPTableGenerator
+
+# generate metrics for EKF
+siap_averages_EKF = {metrics_EKF.get(metric) for metric in metrics_EKF
+                     if metric.startswith("SIAP") and not metric.endswith(" at times")}
+siap_time_based_EKF = {metrics_EKF.get(metric) for metric in metrics_EKF if metric.endswith(' at times')}
+
+# generate metrics for UKF
+siap_averages_UKF = {metrics_UKF.get(metric) for metric in metrics_UKF
+                     if metric.startswith("SIAP") and not metric.endswith(" at times")}
+siap_time_based_UKF = {metrics_UKF.get(metric) for metric in metrics_UKF if metric.endswith(' at times')}
+
+# generate metrics for particle filter
+siap_averages_P = {metrics_P.get(metric) for metric in metrics_P
+                   if metric.startswith("SIAP") and not metric.endswith(" at times")}
+siap_time_based_P = {metrics_P.get(metric) for metric in metrics_P if metric.endswith(' at times')}
+
+# generate metrics for ESIF
+siap_averages_ESIF = {metrics_ESIF.get(metric) for metric in metrics_ESIF
+                      if metric.startswith("SIAP") and not metric.endswith(" at times")}
+siap_time_based_ESIF = {metrics_ESIF.get(metric) for metric in metrics_ESIF if metric.endswith(' at times')}
+
+# %%
+_ = SIAPTableGenerator(siap_averages_EKF).compute_metric()
+print("\n\nSIAP metrics for EKF:")
+
+# %%
+# _ = SIAPTableGenerator(siap_averages_UKF).compute_metric()
+# print("\n\nSIAP metrics for UKF:")
+
+# %%
+# _ = SIAPTableGenerator(siap_averages_P).compute_metric()
+# print("\n\nSIAP metrics for particle filter:")
+
+
+# %%
+# _ = SIAPTableGenerator(siap_averages_ESIF).compute_metric()
+# print("\n\nSIAP metrics for ESIF:")
```

Comment:
I've changed it so that metrics only show for the EKF, ie I removed all metric generation code for the Particle Filter, UKF, and ESIF, reasoning that the user can copy and slightly change the EKF code. Makes the bottom of the example look a bit cleaner in my opinion